core/hw/pvr/ta_ctx.h

```diff
@@ -1,7 +1,30 @@
 #pragma once
+#include <malloc.h>
+#include <stdlib.h>
 #include "ta.h"
 #include "pvr_regs.h"
 
+// helper for 32 byte aligned memory allocation
+inline void* aligned_malloc(size_t size, size_t align)
```

Comment:
Judging from the use, shouldn't this be `(align, size)`?

Reply:
You are absolutely right, sir.  A tiny reservation with massive padding...  Amazingly it still builds and runs (tested remotely and headless so no idea of the on-screen result).